### DISCUSSION TITLE
Automerge scheduling for GitHub Merge Queues

### DISCUSSION THREAD

**Discussion by travisshivers** [renovatebot/renovate#29899] [Suggest an Idea]
### Tell us more.

Using an automerge schedule doesn't work with GitHub merge queues since they require platformAutomerge. However, this means that it will try to automerge immediately ignoring the schedule.

Perhaps something that could make renovate only actually enable the platformAutomerge on the PR during the automerge window.

The automerge schedule for renovate is quite useful and I have a longer automerge schedule compared to the normal schedule such that it will keep the PR up-to-date with new versions for a human to merge sooner if they want.

**Comment by rarkins** [comment]
Converted to issue: https://github.com/renovatebot/renovate/issues/29937
